server/graphql/v2/mutation/RootMutations.ts

```diff
@@ -67,6 +67,10 @@ export default {
         type: GraphQLBoolean,
         description: 'Specify whether the account is a trusted host',
       },
+      isTwoFactorAuthEnabled: {
+        type: GraphQLBoolean,
+        description: 'Specify whether 2FA should be enabled',
```

Comment:
In practice, we can't enable 2FA through this arg

```suggestion
        description: 'Set this to false to disable 2FA. Other values have no effect.',
```